README.md

```diff
@@ -15,11 +15,26 @@ For detailed requirements and install instructions see [irkernel.github.io](http
 We provide Windows and Mac OS X binary packages of all the needed packages:
 
 ```r
-install.packages(c('rzmq','repr','IRkernel','IRdisplay'),
+install.packages(c(repr','IRkernel','IRdisplay'),
```

Comment:
I guess it will be once we do a new release. It's not quite clear whether README should refer to the development version or the released version. But I guess we should do a new release soon (I feel like I'm always saying that ;-).